files/image_config/rsyslog/rsyslog.d/00-sonic.conf.j2

```diff
@@ -47,6 +47,12 @@ if $msg startswith " gnmi-native" then {
     stop
 }
 
+## bmp rules
```

Comment:
No need to add standalone logging file

Reply:
will raise new PR for fixing this later.